browser/src/robocorp/browser/cli.py

```diff
@@ -1,10 +1,25 @@
 # ruff: noqa: E501
 import argparse
+import logging
 import sys
 
 from ._engines import install_browser
 from ._types import BrowserEngine, InstallError
 
+log = logging.getLogger(__name__)
+
+# Use certificates from native storage (if `truststore` installed)
+if sys.version_info >= (3, 10):
+    try:
+        import truststore  # type: ignore
+
+        truststore.inject_into_ssl()
+        log.info(
+            "Truststore injection done, using system certificate store to validate HTTPS."
+        )
+    except ModuleNotFoundError:
+        log.warning("Dependency `truststore` is not installed!")
+
```

Comment:
This looks good!

Do you mind if we put this into a function in the _cli.py_ of the **tasks** package and import it from there and just call the function here? Recommending this because of the following reasons:
1. We need this change (including logging) [here](https://github.com/robocorp/robocorp/blob/master/tasks/src/robocorp/tasks/cli.py#L16-L23) as well.
2. DRY will help you here in case later on we need to come back to this and further fix/improve it. (as it is used from one single place)
    - **browser** already depends on **tasks** if you look in the _pyproject.toml_.